### PR TITLE
world-information web-app

### PR DESCRIPTION
**Pull request by SabaAhmad404**
This is my capstone project my capstone topic was world's information Api in this project
1) i fetched the countries data
2) fetched each country's information like population, language, border,
3) design the app using the CSS
4) here is my loom video 
kindly review my project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,4 +123,6 @@ Give a ⭐️ if you like this project!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (FrederickMih) [inline README.md]: - [x] The content of the **Acknowledgments** section in your README file is missing. Also, as stated in the requirements the README file should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly add the content of this section as in the README template with acknowledgments to the designer.

File: src/components/homepage.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+import { IoSearch } from 'react-icons/io5';
+import { countriesFetch, continents } from '../redux/countrySlice';
+import worldmap from '../assets/worldmap.svg';
+import Navbar from './Nav';
+import CountrySection from './countrysection';
+
+const Home = () => {
```
Comment (FrederickMih) [inline src/components/homepage.js:10]: - [x] Hi @SabaAhmad404, let's kindly note that this is a _mobile web app_ and we are requested to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements.

**Screenshots:**

| Original | Yours |
| --- | --- |
| CaptureO | Capture |

**NB:** Having a nice design increases your chances for a job as recruiters will be checking your GitHub projects.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi 🙋 @SabaAhmad404, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights 🟢 🟢 🟢 

- Redux has been used for state management ✔️ 
- PR is professional ✔️ 
- Linters checks passed ✔️


## Suggested changes

- [x] **Video Presentation:** Great job presenting the features of your app 👍  Please kindly highlight the piece of code you finds interesting or proud of as stated in the project's requirements. You'll probably need to record another short video for this aspect.
- [x] **Testing** Some tests are not passing as  a result of the issue flagged on the screenshot below:

Capture

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @SabaAhmad404 , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [x] Good job re-recording the presentation video, but unfortunately you did not highlight a piece of code you are proud of. Please kindly re-record the video and highlight a piece of code you are proud of making. Also, do not forget to update the link in the PR description.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by SabaAhmad404** [comment]
@AnselemOdims  ok thank you 😊

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Saba, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦   🏃‍♀️ 

## Highlights 👍  
- Good job for improvement so far!

## Required Changes ♻️
- [ ] Good job with project presentation, however, you missed some steps in your video presentation, as mentioned in the previous review. Your video presentation only showed the feature aspects of your app, however, highlights of some portions of your code are missing in the presentation. Kindly ensure that the video contains a highlight about a piece of the code (possibly from your code editor) that you find interesting or that you were proud of making.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- You used incorrect git-flow/ GitHub flow for this project. Kindly note that a capstone project worked on by a single contributor should have merge request flow **from feature into development**. Kindly note this important step for subsequent projects.
- Kindly include appropriate instructions under the Getting Started section of your ReadMe, especially on how to _run_ your app and _run test_ cases for this project.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @SabaAhmad404 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Good presentation demo
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SabaAhmad404** [comment]
@omarsalem7 
thank you that was exhausted process damn

**Pull request merged**